ghost/email-service/lib/BatchSendingService.js

```diff
@@ -513,6 +529,24 @@ class BatchSendingService {
                 tiers
             };
         });
+
+        const BATCH_SIZE = this.#sendingService.getMaximumRecipients();
+        if (mappedMemberLikes.length > BATCH_SIZE) {
+            logging.warn(`Batch ${batchId} has ${mappedMemberLikes.length} members, but the sending service only supports ${BATCH_SIZE} members per batch.`);
+
+            const _debug = require('@tryghost/debug')._base;
+            try {
+                _debug.enable('ghost-query');
+                await this.#models.EmailRecipient.findAll({filter: `batch_id:${batchId}`, withRelated: ['member', 'member.stripeSubscriptions', 'member.products']});
+            } finally {
+                // @NOTE: disables all debugged modules, there's no
+                //        way to disable just one module.
+                //        The behavior rare enough and is acceptable to catch the live issue
+                _debug.disable();
```

Comment:
@naz Sorry this is just probably my lack of context on this, but what's the effect of `_debug.disable()` ?

Reply:
Above we enable `_debug.enable('ghost-query');` debug outputs for db queries, with hopes to fetch what exact queries were run at the time when size of the batch is larger than we expect. the `_debug.disable()` statement stops those outputs after the query produced by `await this.#models.EmailRecipient.findAll()` finishes execution

Reply:
Ah perfect, thanks